topics/variant-analysis/tutorials/baculovirus-isolate-variation/tutorial.md

```diff
@@ -0,0 +1,851 @@
+>    > * Splitting `DPR` data into separate allele counts.   
+>    > * Calculating the relative allele frequencies (`REL.ALT`) for each allele.  
+>    > * Filtering out relative frequencies below a threshold of 0.05 and storing the result in `REL.ALT.0.05`.  
+>    {: .comment}
+>
+{: .hands_on}
+
+
+The output table is complex and shows the relative frequency (`REL.ALT`) for each of the three alternative alleles/nucleotides (`ALT1`, `ALT2` and `ALT3`) in each position (`POS`) and sequenced CpGV isolate (`SAMPLE`). In column `REL.ALT.0.05`, values of the `REL.ALT < 0.05` were set to 0 to set a threshold. We will see later why this is sometimes important.
+
+Below is the table with selected relevant columns only. `REL` and `ALT` show the reference and alternative nucleotide, respectively.    
+
+| CHROM  | POS | REF | ALT | SAMPLE      | DP  | DPR         | ALLELE | DPR.ALLELE | REL.ALT    | REL.ALT.0.05 |
+|--------|-----|-----|-----|-------------|-----|-------------|--------|------------|------------|--------------|
+| CpGV-M | 246 | C   | T   | SRR31589146 | 886 | 107,773,6,0 | ALT1   | 773        | 0.87246    | 0.873446     |
+| CpGV-M | 246 | C   | G   | SRR31589146 | 886 | 107,773,6,0 | ALT2   | 6          | 0.00677201 | 0            |
+| CpGV-M | 246 | C   | A   | SRR31589146 | 886 | 107,773,6,0 | ALT3   | 0          | 0          | 0            |
+| CpGV-M | 246 | C   | T   | SRR31589147 | 881 | 5,875,1,0   | ALT1   | 875        | 0.99319    | 0.994305     |
+| CpGV-M | 246 | C   | G   | SRR31589147 | 881 | 5,875,1,0   | ALT2   | 1          | 0.00113507 | 0            |
+| CpGV-M | 246 | C   | A   | SRR31589147 | 881 | 5,875,1,0   | ALT3   | 0          | 0          | 0            |
+| CpGV-M | 246 | C   | T   | SRR31589148 | 935 | 800,133,1,1 | ALT1   | 133        | 0.142246   | 0.142398     |
+| CpGV-M | 246 | C   | G   | SRR31589148 | 935 | 800,133,1,1 | ALT2   | 1          | 0.00106952 | 0            |
+| CpGV-M | 246 | C   | A   | SRR31589148 | 935 | 800,133,1,1 | ALT3   | 1          | 0.00106952 | 0            |
+| CpGV-M | 246 | C   | T   | SRR31679023 | 839 | 42,797,0,0  | ALT1   | 797        | 0.94994    | 0.950296     |
+| CpGV-M | 246 | C   | G   | SRR31679023 | 839 | 42,797,0,0  | ALT2   | 0          | 0          | 0            |
+| CpGV-M | 246 | C   | A   | SRR31679023 | 839 | 42,797,0,0  | ALT3   | 0          | 0          | 0            |
+
+
+## Replace SRA Names with Virus Abbreviations
+
+One thing that stands out are the SAMPLE names, which were taken automatically from the NCBI SRA datasets. Since it is difficult to remember which virus isolate is behind which SRA number, we can replace the accession numbers with proper names. This makes the table even easier to read and later we can use the information directly to display the SNV positions. 
```

Comment:
```suggestion
One thing that stands out is the SAMPLE names, which were taken automatically from the NCBI SRA datasets. Since it is difficult to remember which virus isolate is behind which SRA number, we can replace the accession numbers with proper names. This makes the table even easier to read and later we can use the information directly to display the SNV positions. 
```